gbfs.md

```diff
@@ -1137,6 +1139,60 @@ Field Name | REQUIRED | Type | Defines
 }
 ```
 
+### vehicle_availability.json
+
+*(added in v3.1-RC2)*
+
+Describes the future availability of each vehicle. Useful for systems that allow vehicles to be reserved in advance (e.g. carsharing, cargo bike share, etc). This file is OPTIONAL for station based (docked) vehicles. Not supported for free floating (dockless) vehicles. Data returned SHOULD be as close to realtime as possible, but in no case should it be more than 5 minutes out-of-date. See [Data Latency](#data-latency).<br/>The following fields are all attributes within the main `data` object for this feed.
+
+Field Name | REQUIRED | Type | Defines
+---|---|---|---
+`vehicles` | REQUIRED | Array&lt;Object&gt; | Contains one object per vehicle.
+`vehicles[].vehicle_id` | REQUIRED | ID | Identifier of a vehicle. The `vehicle_id` identifier MUST be rotated to a random string after each trip to protect user privacy *(as of v2.0)*. Use of persistent vehicle IDs poses a threat to user privacy. The `vehicle_id` identifier SHOULD only be rotated once per trip.
```

Comment:
Probably these vehicle_ids should be the same as in `vehicle_status.json`. 
That would make vehicle_status.json required when using vehicle_availability.json, but I don't think anyone would want to display only the availability of the vehicle, but not the vehicle's location.
So I think, we could write:

```suggestion
`vehicles[].vehicle_id` | REQUIRED | ID | Unique identifier of a vehicle as defined in [vehicle_status.json](#vehicle_statusjson).
```